### PR TITLE
[DDW-600] Progress bar animation and positioning issues

### PR DESCRIPTION
**Pull request by daniloprates**
This PR fixes animation and positioning issues in the progress bar

## Todos

- [x] Fix progress bar wrong alignment (Linux only)
- [x] Fix progress bar glitches in the animation
- [JIRA](https://jira.iohk.io/browse/DDW-600) 
- [Storybook](http://localhost:6006/?path=/story/news-overlays--update#themeName=White&localeName=English&osName=Linux)
  - To test the misalignment issue on Storybook, set the following parameters:
    - Scenario: Download complete
    - isLinux: true
    - isWaitingToQuitDaedalus: true
  - To reproduce the issue, add the class `AppUpdateOverlay_actions` to the parent div of `AppUpdateOverlay_progressBar`
- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1615838673300500)

### Alignment issue

**Incorrect**
Alignment - Incorrect

**Correct**
Alignment - Correct

### Animation issue

**Incorrect**
Animation - Incorrect

**Correct**
Animation - Correct

---

## Testing Checklist

- [ ] On Linux, the progress bar should not be misaligned
- [ ] The progress bar animation should not have glitches

---
 ## Test scenarios

**Scenario 1 - Validate alignment of the progress bar in Linux**
```gherkin
Given that I am running Daedalus on Linux
And an update is pushed through the newsfeed
When I start the installation
And check the progress bar
Then I can see the progress bar is aligned with the text area.
```

**Scenario 2 -  Validate progress bar animation**
```gherkin
Given that I am on |Screen|
And the progress bar is loading
When I check the animation
Then I can see the pattern is uniform across the animation.

|Screen|
Update available (download)
Update available (install - Linux only)
Voting registration - 3rd step
```
---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [ ] In English
  - [ ] In Japanese
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Review by nikolaglumac — CHANGES_REQUESTED** [review]
@daniloprates the animation on the Voting registration still looks broken. Can you please check that? 🙏 


https://user-images.githubusercontent.com/376611/111268912-34f4ad00-862e-11eb-86fd-37df18b83116.mp4

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Changelog
 ### Fixes
 
 - Fixed delay on loading stake pool list after switching the SMASH server ([PR 2447](https://github.com/input-output-hk/daedalus/pull/2447))
+- Fixed animation and positioning issues in the progress bar ([PR 2458](https://github.com/input-output-hk/daedalus/pull/2458))
```
Comment (tomislavhoracek) [inline CHANGELOG.md:9]: @daniloprates please move a log to the top 😄

Comment (tomislavhoracek) [inline CHANGELOG.md:9]: @daniloprates please move a log to the top.

**Comment by gabriela-ponce** [comment]
@daniloprates Overall, the fixes look good (the alignment in Linux included). The progress bar in voting still has the animation off, as Nikola [mentioned](https://github.com/input-output-hk/daedalus/pull/2458#pullrequestreview-612905636). I only checked this item in Storybook, though (registry for voting is disabled on most networks and closed on Mainnet).

File: CHANGELOG.md (hunk 1 above)
Comment (daniloprates) [reply CHANGELOG.md:8]: @tomislavhoracek done ✅

**Comment by daniloprates** [comment]
> @daniloprates the animation on the Voting registration still looks broken. Can you please check that? 🙏
> 
>  voting-registration.mp4

@nikolaglumac done ✅

**Review by tomislavhoracek — APPROVED** [review]
💯

**Review by gabriela-ponce — APPROVED** [review]
Looks good 👍

**Review by miorsufianiohk — APPROVED** [review]
LGTM. Tested on 17157 testnet and mainnet. Great work @daniloprates 👍

**Review by nikolaglumac — APPROVED** [review]
Great work! 👌

**Pull request merged**